drydock_backups/templates/drydock-backups/.gitignore

```diff
@@ -0,0 +1 @@
+!/build
```

Comment:
I would prefer changing the [global gitignore](https://github.com/eduNEXT/drydock/blob/main/.gitignore#L11)